spanner/api/Spanner.Samples/AddColumnAsync.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_add_column]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class AddColumnAsyncSample
+{
+    public async Task AddColumnAsync(string projectId, string instanceId, string databaseId)
+    {
+        // Initialize request argument(s).
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
```

Comment:
Here and in the files below, there is a reason why this string was line-wrapped in Program.cs. It's because if the line is too long, you can't see it all when the sample loads on the page (e.g. on laptop screens) and then you have to scroll right to see it. I don't know if we have a limit specified somewhere but I would cut off line lengths to 100.

@amanda-tarafa let us know if you have any guidance on this.